code/index.html

```diff
@@ -9,39 +9,79 @@
       href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
       rel="stylesheet"
     />
+    <link rel="shortcut icon" href="#" />
   </head>
+
   <body>
     <aside class="question-section">
-      <button id="restart" class="outlined-button">RESTART</button>
-
-      <img
-        class="guess-who-icon"
-        src="images/guess-who-bubble.png"
-        alt="Guess Who"
-      />
+      <button id="restart" class="filled-button">RESTART</button>
+      <div class="guess-who">
+        <h1>?</h1>
+        <div class="guess-who__spans">
+          <span>Guess</span>
+          <span>Who!</span>
+        </div>
+      </div>
 
-      <h1>Does the person have</h1>
+      <h2>Does the champion have:</h2>
 
       <select id="questions">
-        <optgroup label="hair">
+        <!-- Hair color  -->
```

Comment:
Your section with options are super clear, its so nice to follow the code. It is very many alternatives, and it could have been a lot to process when reading as an "outsider" but I follow it without any trouble.